### ISSUE TITLE
A error happen when add a new table to osquery

### ISSUE THREAD

**Issue by foreversunyao** [sidorares/mysql-osquery-proxy#1]
@sidorares 

```
I add a new table to osquery and recompile osqueryi, when I only use osqueryi(client), it can work, but when I use mysql-osquery-proxy, it does not work, did I miss some steps ? I already replace osqueryi to new version
```

**osqueryi:**
Using a virtual database. Need help, type '.help'
osquery> select \* from fio;
+--------------------------+----------------------+---------------+------------+------------+-------------+------------+-----------+------+
| fio_product              | part_number          | serial_number | fm_version | dr_version | rsvd_status | rated_prbw | cur_tempr | size |
+--------------------------+----------------------+---------------+------------+------------+-------------+------------+-----------+------+
| xxxx | xxx |xxx     | 7.1.17     | 3.2.14     | Healthy     | 2,57.39    | 53.15     | 410  |
+--------------------------+----------------------+---------------+------------+------------+-------------+------------+-----------+------+

**use mysql-osquery-proxy:**
ERROR 1064 (_____): Error running query: no such table: fio

**Comment by sidorares** [comment]
hm, `Error running query: no such table: fio` is answer from osquery itself, maybe there is some action required so that clients aware about new tables? Are you able to see "standard" built-in tables?

**Comment by foreversunyao** [comment]
@sidorares 
    Yes , we can see table fio by .table command, dose mysql-osquery-proxy support the new table added to base osquery version? or is there any example ?

```
Using a virtual database. Need help, type '.help'
```

osquery> .tables
  => acpi_tables
  => apt_sources
  => arp_cache
  => authorized_keys
  => block_devices
  => carbon_black_info
  => chrome_extensions
  => cpu_time
  => cpuid
  => crontab
  => deb_packages
  => device_file
  => device_hash
  => device_partitions
  => disk_encryption
  => dns_resolvers
  => etc_hosts
  => etc_protocols
  => etc_services
  => file
  => file_events

## **=> fio**

**Comment by sidorares** [comment]
not sure, this module basically talks osquery using thrift protocol ( [.query method](https://github.com/facebook/osquery/blob/60bbf129af9dc3742fb24689746d74c4fc73660d/osquery.thrift#L89-L91) ) on one side and exposes as mysql server on another side
